src/Shamsheer.Messenger.Api/Controllers/ChannelAssets/ChannelAssetsController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Shamsheer.Service.Configurations;
+using Shamsheer.Service.Interfaces.ChannelAssets;
+
+namespace Shamsheer.Messenger.Api.Controllers.ChannelAssets;
+
+[Route("api/[controller]")]
+[ApiController]
+public class ChannelAssetsController : ControllerBase
```

Comment:
BaseController degan class'da Route'larni yozib olib chiqib ketgandigu? O'shandan voris olishiz kerakmasmi Ziyodulloh aka?

Reply:
ha to'g'irlab qo'yaman , tushunarlik :)

Reply:
no problem👍